cdap-ui/package.json

```diff
@@ -115,6 +115,7 @@
     "react-dropzone": "3.6.0",
     "react-file-download": "0.3.1",
     "react-helmet": "3.1.0",
+    "react-paginate": "^4.1.1",
```

Comment:
Yes. We need to rebuild for this and for `react-timeago` that was added recently.